README.md

```diff
@@ -16,6 +16,39 @@ the `alpha` branch.
 
 Please note that this specification is currently under heavy construction.
 
+## Local Devnet Setup
+
+You can spin up a local devnet via `docker-compose`. For convenence, we have defined `make` targets to start and stop the devnet with a single command. To run the devnet, you will need `docker` and `docker-compose` installed. Then, as a precondition, make sure that you have compiled the contracts by `cd`ing into `packages/contracts` and running `yarn` followed by `yarn build`. You'll only need to do this if you change the contracts in the future.
+
+Then, run the following:
+
+```bash
+make devnet-up # starts the devnet
+make devnet-down # stops the devnet
+make devnet-clean # removes the devnet by deleting images and persistent volumes
+```
+
+L1 is accessible at `http://localhost:8545`, and L2 is accessible at `http://localhost:8546`. Any Ethereum tool - Metamask, `seth`, etc. - can use these endpoints. Note that you will need to specify the L2 chain ID manually if you use Metamask. The devnet's L2 chain ID is 901.
```

Comment:
8546 is commonly used for websocket RPC. (some clients have it on the same port, some don't, iirc)
Why not just use something like `9545` for L2?

Reply:
Sure, this is easy to change.